exasol_transformers_extension/deployment/language_container_deployer_cli.py

```diff
@@ -51,7 +51,9 @@ def update_parameter(parameter_name: str, formatter: str) -> None:
             param_formatter = ctx.params.get(parameter_name, formatter)
             if param_formatter:
                 # Enclose in double curly brackets all other parameters in the formatting string,
-                # to avoid the missing parameters' error.
+                # to avoid the missing parameters' error. Below is an example of a formatter string
+                # before and after applying the regex, assuming the current parameter is 'version'.
+                # 'something-with-{version}/tailored-for-{user}' => 'something-with-{version}/tailored-for-{{user}}'
```

Comment:
did you actually change the wrong parameter

Reply:
The pattern in this example would be `\{(?!version\})\w+\}` and it matches all substrings `{...}` that are not `{version}` .